uiowa/uiowa
# Feature webform styling

Resolves #4954 
Resolves https://github.com/uiowa/uiowa/issues/5002
Resolves https://github.com/uiowa/uiowa/issues/3486

# How to test

**For tippie form example**
1. Pull down tippie.students
`blt ds --site=students.tippie.uiowa.edu`
2. visit the page: https://tippiestudents.uiowa.ddev.site/undergraduates/get-involved/tippie-top-21/submit-tippie-top-21-nomination
3. Check to see if it matches the notes referenced in #4954.

**To test all available form fields**
1. Pull down sandbox prod
`blt ds --site=sandbox.uiowa.edu`
2. Test the page https://sandbox.uiowa.ddev.site/form

**Test bug fixes for https://github.com/uiowa/uiowa/issues/5002 and https://github.com/uiowa/uiowa/issues/3486**
- Add a text field and for the Title use "Submitter's Name" or anything with the word submit.
- Set a fixed width for a text field and verify that the field width is not being set to 100% if there is a value besides 60 in the field.
